domains/sunnyychi.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "A personal portfolio website",
+  "repo": "https://github.com/sunnyychi/Website",
+  "owner": {
+    "username": "sunnyychi",
+    "email": "sunnyychi@lona-development.org"
+  },
+  "record": {
+    "CNAME": "sunnyychi.glitch.cme"
```

`cme` is not a valid TLD, did you mean `me`?

```suggestion
    "CNAME": "sunnyychi.glitch.me"
```

---

Yes, my bad